Palindrome Number/main.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace DefaultNamespace
+{
+    public class main
+    {
+        public bool IsPalindrome(int x)
+        {
+            if (x < 0 || (x!=0 && x % 10==0)) return false;
```

Comment:
@ziadadeela this condition is true when x is a negative number or ends with zero, eg. 10 can't be a palindrome